Gemfile

```diff
@@ -1,17 +1,19 @@
-source 'http://rubygems.org'
+source 'https://rubygems.org'
 
 ruby '2.4.9'
```

Comment:
This may be unrelated, but we have newer Ruby versions now.

Support for Ruby 2.4 has ended!
https://www.ruby-lang.org/en/news/2020/04/05/support-of-ruby-2-4-has-ended/

Reply:
Little by little. Let's finish with Rails ;)

Reply:
👌